### PR TITLE
executor: Improve the aesthetics of code review 

### PR DESCRIPTION
**Pull request by kangxiaoning**
Improve the aesthetics of code review in executor/grant_test.go.

### CODE REVIEW

File: executor/grant_test.go [hunk 1]
```diff
@@ -176,26 +176,26 @@ func (s *testSuite) TestColumnScope(c *C) {
 
 func (s *testSuite) TestIssue2456(c *C) {
 	tk := testkit.NewTestKit(c, s.store)
-	tk.MustExec("CREATE USER 'dduser'@'%' IDENTIFIED by '123456';")
-	tk.MustExec("GRANT ALL PRIVILEGES ON `dddb_%`.* TO 'dduser'@'%';")
-	tk.MustExec("GRANT ALL PRIVILEGES ON `dddb_%`.`te%` to 'dduser'@'%';")
+	tk.MustExec("CREATE USER 'dduser'@`%` IDENTIFIED by '123456';")
```
Comment (winkyao) [inline executor/grant_test.go]: why change this?

Comment (yangwenmai) [reply executor/grant_test.go]: You modified it wrong. You should modify the quotation marks for the SQL statement instead of the quotation marks for %.

Comment (kangxiaoning) [reply executor/grant_test.go]: @winkyao @yangwenmai I don't understand why changing the backquote to single quotes would report the following error. So keep the original backquote.

```
FAIL: grant_test.go:177: testSuite.TestIssue2456

grant_test.go:180:
    tk.MustExec(`GRANT ALL PRIVILEGES ON 'dddb_%'.* TO 'dduser'@'%';`)
/Users/marong/go/src/github.com/pingcap/tidb/util/testkit/testkit.go:160:
    tk.c.Assert(err, check.IsNil, check.Commentf("sql:%s, %v, error stack %v", sql, args, errors.ErrorStack(err)))
... value *errors.Err = &errors.Err{message:"", cause:(*errors.errorString)(0xc4201c2d40), previous:(*errors.Err)(0xc4205d2730), file:"/Users/marong/go/src/github.com/pingcap/tidb/util/testkit/testkit.go", line:134} ("line 1 column 32 near \".* TO 'dduser'@'%';\" (total length 51)")
... sql:GRANT ALL PRIVILEGES ON 'dddb_%'.* TO 'dduser'@'%';, [], error stack line 1 column 32 near ".* TO 'dduser'@'%';" (total length 51)
```

Comment (winkyao) [reply executor/grant_test.go]: Just keep the original code is ok:

```
tk.MustExec("CREATE USER 'dduser'@'%' IDENTIFIED by '123456';")
tk.MustExec("GRANT ALL PRIVILEGES ON `dddb_%`.* TO 'dduser'@'%';")
tk.MustExec("GRANT ALL PRIVILEGES ON `dddb_%`.`te%` to 'dduser'@'%';")
```

Comment (kangxiaoning) [reply executor/grant_test.go]: @winkyao The code have changed to original, and `make dev` successfully on local, but check failed on github.

Comment (kangxiaoning) [reply executor/grant_test.go]: @winkyao `executor/grant_test.go` have been changed to original, thx.

**Comment by ngaut** [comment]
PTAL @yangwenmai

**Comment by ngaut** [comment]
LGTM

**Pull request merged**